### PR TITLE
Revert namespace

### PR DESCRIPTION
**Pull request by AntonReinhard**
Reverts the name space changes and removes QEDprocesses from the dependencies.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,20 +1,13 @@
 name = "QEDevents"
 uuid = "fc3ce04a-5be5-4f3a-acff-eceaab723759"
-authors = [
-    "Uwe Hernandez Acosta <u.hernandez@hzdr.de>",
-    "Simeon Ehrig",
-    "Klaus Steiniger",
-    "Tom Jungnickel",
-    "Anton Reinhard",
-]
+authors = ["Uwe Hernandez Acosta <u.hernandez@hzdr.de>", "Simeon Ehrig", "Klaus Steiniger", "Tom Jungnickel", "Anton Reinhard"]
```
Comment (szabo137) [inline Project.toml:3]: Again? Let's see who wins this battle 🥷

File: src/patch_QEDbase.jl [hunk 2]
```diff
@@ -4,6 +4,3 @@
 # See https://github.com/QEDjl-project/QEDbase.jl/issues/69
 struct UnknownDirection <: QEDbase.ParticleDirection end
 Broadcast.broadcastable(d::UnknownDirection) = Ref(d)
-
-is_incoming(::UnknownDirection) = false
```
Comment (szabo137) [inline src/patch_QEDbase.jl:8]: Why removing just this? Is this patched upstream? Why not removing the whole thing?

**Review by szabo137 — APPROVED** [review]
LGFM. Thanks for this 🙏

File: src/patch_QEDbase.jl (hunk 2 above)
Comment (AntonReinhard) [reply src/patch_QEDbase.jl:8]: It's an open issue, so not in QEDbase yet as far as I'm aware

File: Project.toml (hunk 1 above)
Comment (AntonReinhard) [reply Project.toml:3]: Maybe we should figure out a way to have this consistently formatted 🙃

**Pull request merged**